Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1532,4 +1536,103 @@ namespace Babylon
     {
         return Napi::Value::From(info.Env(), m_graphicsImpl.GetHardwareScalingLevel());
     }
+
+    Napi::Value NativeEngine::CreateImageBitmap(const Napi::CallbackInfo& info)
+    {
+        const auto env = info.Env();
+        if (!info[0].IsArray())
+        {
+            throw Napi::Error::New(env, "CreateImageBitmap first parameter must be an array.");
+        }
+        const auto parameters = info[0].As<Napi::Array>();
+        if (!parameters.Length())
+        {
+            throw Napi::Error::New(env, "CreateImageBitmap parameters are empty.");
+        }
+
+        const auto blob = parameters[0u].As<Napi::Array>();
```

Comment:
Is the intention to support everything that browser can support? Why pass in a blob at all? If we only want to support arraybuffer, do the blob checking in JS and pass in an arraybuffer.